awx/ui_next/src/screens/Job/JobOutput/JobOutput.jsx

```diff
@@ -85,8 +132,21 @@ class JobOutput extends Component {
   }
 
   componentDidMount() {
+    const { job } = this.props;
+    const { remoteRowCount } = this.state;
     this._isMounted = true;
     this.loadJobEvents();
+
+    const updateRemoteRowCount = throttle(
+      count => this.setState({ remoteRowCount: count }),
+      5000
+    );
+
+    connectJobSocket(job, data => {
+      if (data.counter && data.counter > remoteRowCount) {
```

Comment:
All we actually need from the websocket events is to look at their counters so we can keep track of the total number of events expected to exist on the backend somewhere (now or eventually)